plugwise/helper.py

```diff
@@ -867,26 +843,43 @@ def _appliance_measurements(
                 if new_name := getattr(attrs, ATTR_NAME, None):
                     measurement = new_name
 
-                data[measurement] = appl_p_loc.text  # type: ignore [literal-required]
                 # measurements with states "on" or "off" that need to be passed directly
-                if measurement not in ("select_dhw_mode"):
-                    data[measurement] = format_measure(  # type: ignore [literal-required]
+                if measurement == "select_dhw_mode":
```

Comment:
Should we still support lower python3 versions, if not this would be a great candidate for match+case

Reply:
Implemented.